packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_def.hpp

```diff
@@ -409,7 +409,7 @@ compute ()
   const local_ordinal_type numCols = A.numCols ();
   const local_ordinal_type annz    = A.nnz ();
 
-  typename crs_matrix_type::local_matrix_type::row_map_type newptr ("ptr", ptr.dimension_0 ());
+  typename crs_matrix_type::local_matrix_type::row_map_type::non_const_type newptr ("ptr", ptr.dimension_0 ());
   typename crs_matrix_type::local_matrix_type::index_type::non_const_type newind ("ind", ind.dimension_0 ());
   typename crs_matrix_type::local_matrix_type::values_type newval ("val", val.dimension_0 ());
 
```

Comment:
@sthomas61 Please see my previous comment.  Just use KokkosSparse::CrsMatrix directly.  You can test whether it is initialized by checking the dimension of its row offsets array.